### DISCUSSION TITLE
Canaries

### DISCUSSION THREAD

**Discussion by alitair** [birdnet-team/BirdNET-Analyzer#467] [General]
Using a species list to just identify canaries, e.g.

Crithagra albogularis_White-throated Canary
Crithagra atrogularis_Black-throated Canary
Crithagra buchanani_Southern Grosbeak-Canary
Crithagra dorsostriata_White-bellied Canary
Crithagra flaviventris_Yellow Canary
Crithagra mozambica_Yellow-fronted Canary
Crithagra scotops_Forest Canary
Crithagra sulphurata_Brimstone Canary
Culicicapa ceylonensis_Gray-headed Canary-Flycatcher
Culicicapa helianthea_Citrine Canary-Flycatcher
Serinus alario_Black-headed Canary
Serinus canaria_Island Canary
Serinus canicollis_Cape Canary
Serinus flavivertex_Yellow-crowned Canary

and then giving analyze.py sounds of canaries in a cage delivers the result that canary song and chirps are undetected as a canary, but cage noise is positively identified as a canary. not good.

**Comment by NickleDave** [comment]
This is probably a case of domain shift since BirdNET is trained on ARU data, no?
Could you fine tune on a [canary dataset](https://datadryad.org/stash/dataset/doi:10.5061/dryad.xgxd254f4)?
